docs/reference-guide/components/view-jpa.md

```diff
@@ -62,6 +62,7 @@ polyflow.view.jpa:
   stored-items: task, data-entry, process-instance, process-definition
   payload-attribute-level-limit: 2
   include-correlated-data-entries-in-data-entry-queries: false
+  process-old-events: false
```

Comment:
I think `process-outdated-events` would be slightly better wording here. It emphasizes that there is actually a newer event and not just that the event is "old" as in "around for a while"